Applets/UI/TaskView/TaskView.swift

```diff
@@ -20,12 +20,13 @@ struct TaskView: View {
             ZStack(alignment: .leading) {
                 Rectangle()
                     .frame(width: 370, height: 25)
+                    .modifier(FilledCellModifier(percentOfDone: viewModel.goal.fractionOfDone))
                     .foregroundColor(.gray.opacity(0.30))
                     .cornerRadius(20)
-                Rectangle()
-                    .frame(width: 70, height: 25)
-                    .foregroundColor(.accentColor)
-                    .cornerRadius(20)
+//                Rectangle()
```

Comment:
Please, remove all commented and unused code before creating pull request, because at some point our code will be really messy